assets/src/edit-story/components/keyboard/index.js

```diff
@@ -50,7 +50,7 @@ const globalRef = createRef();
  * @param {function(KeyboardEvent)} callback
  * @param {Array|undefined} deps
  */
-function useKeyEffect(
+function _useKeyEffect(
```

Comment:
In this codebase we so far moved away from `_`. Let's call this method `useKeyEffectInternal`.

Reply:
Done.